pypogs/hardware_cameras.py

```diff
@@ -139,9 +141,11 @@ def __init__(self, model=None, identity=None, name=None, auto_init=True, debug_f
             self.identity = identity
         if name is not None:
             self.name = name
-        if auto_init and not None in (model, identity):
+        if auto_init and model is not None:
```

Comment:
Check if model is ptgrey, if true must check identity is not None before auto init.